[2022-11-13T17:12:31.350+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v1.create_postgres_table scheduled__2022-11-08T00:00:00+00:00 [queued]>
[2022-11-13T17:12:31.379+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v1.create_postgres_table scheduled__2022-11-08T00:00:00+00:00 [queued]>
[2022-11-13T17:12:31.380+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-13T17:12:31.381+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 7
[2022-11-13T17:12:31.382+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-13T17:12:31.416+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2022-11-08 00:00:00+00:00
[2022-11-13T17:12:31.428+0000] {standard_task_runner.py:55} INFO - Started process 3407 to run task
[2022-11-13T17:12:31.459+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v1', 'create_postgres_table', 'scheduled__2022-11-08T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpmrj09h4e']
[2022-11-13T17:12:31.466+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask create_postgres_table
[2022-11-13T17:12:31.658+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_with_postgres_operator_v1.create_postgres_table scheduled__2022-11-08T00:00:00+00:00 [running]> on host 6a6c6e07e37b
[2022-11-13T17:12:31.828+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=charlie
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v1
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-08T00:00:00+00:00
[2022-11-13T17:12:31.852+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-11-13T17:12:31.868+0000] {sql.py:315} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2022-11-13T17:12:31.873+0000] {postgres.py:96} INFO - NOTICE:  relation "dag_runs" already exists, skipping

[2022-11-13T17:12:31.890+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v1, task_id=create_postgres_table, execution_date=20221108T000000, start_date=20221113T171231, end_date=20221113T171231
[2022-11-13T17:12:31.953+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-13T17:12:32.023+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
